docs/source/osds/scenarios.rst

```diff
@@ -0,0 +1,26 @@
+OSD Scenarios
+=============
+
+lvm
+---
+This OSD scenario uses ``ceph-volume`` to create OSDs from logical volumes and
+is only available when the ceph release is Luminous or greater.
```

Comment:
yeah same here, "Luminous or newer" I think would be consistent and since you aren't using a number, "greater" doesn't correlate to the named-version